Automattic/wp-calypso
# Assembler: Add the Upsell screen

Related to https://github.com/Automattic/wp-calypso/issues/80682

## Proposed Changes

* Added the Upsell screen

| Without resetting custom styles | With resetting custom styles |
| - | - |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/784f7d9a-7617-44ed-a9e9-6d29aea9e4ed) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/b794adc1-8d6d-4207-bb7d-fc75bb440b7d) |

## Testing Instructions



* Go to `/setup?siteSlug=<your_site>` on your free site
* Continue to Design Picker
* Pick “Design your own”
* On the Assembler screen
  * Pick patterns
  * Pick styles
  * Ensure you can see the Upsell screen
  * Toggle the “Reset custom styles”
  * Ensure the global styles won't be applied and you can see the “Edit your content” button now

## Pre-merge Checklist
